src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -395,5 +396,158 @@ export const content: GettingStartedContent = [
 				}
 			]
 		}
+	},
+
+	{
+		id: 'Intermediate',
+		title: localize('gettingStarted.intermediate.title', "Boost your Productivity"),
+		icon: intermediateIcon,
+		description: localize('gettingStarted.intermediate.description', "Optimize your development workflow with these tips & tricks."),
+		content: {
+			type: 'items',
+			items: [
+				{
+					id: 'playground',
+					title: localize('gettingStarted.playground.title', "Redefine your editing skills"),
+					description: localize('gettingStarted.playground.description', "Want to code faster and smarter? Practice powerful code editing feature without creating files or a project."),
```

Comment:
I like framing this as practicing without creating a project, but I also think of the tips in interactive playground as life hacks. Perhaps: "Want to code faster and smarter? Practice these must-know editing hacks in the playground."

Reply:
_Life hacks_ makes some sense, but _hacks_ doesn't seem the right for multi-cursor. Lets try this out as less formal version during the content experiment!